certbot/docs/using.rst

```diff
@@ -802,64 +741,116 @@ configuration file we advise you to make a backup of the file beforehand and tes
 
 .. warning:: Manually modifying files under ``/etc/letsencrypt/renewal/`` can damage them if done improperly and we do not recommend doing so.
 
+.. _references:
```

Comment:
I think we should maybe discuss what we want the final documentation to look like, in terms of a balance of the [four different functions](https://documentation.divio.com/introduction/#the-secret). I think a piecemeal approach to rewriting the docs is going to work great, but it would be good to get a sense for how this "Reference" concept would grow in the documentation and to have a plan around that, as the rewrites happen.

Reply:
Yes, I was hoping tackling this portion of the documentation would get us to make this decisions for technical references and where they should ultimately go. My initial thoughts were
- Each major section has it's own Reference.
Or
- Build one Reference section. 

My main worries with option two, was it getting too monolithic.

Let me know what you think of those options, or offer another take!